src/components/communities/CommunityTab/CommunityTab.tsx

```diff
@@ -0,0 +1,118 @@
+'use client'
+
+import { useState } from 'react'
+import EventCardCommunity from '../EventCardCommunity/EventCardCommunity'
+import CommunityMembers from '../CommunityMembers/CommunityMembers'
+import { OrganizationUserType } from '@/schemas/organizerSchema'
+import { UserType } from '@/schemas/userSchema'
+import { EventCommunityType } from '@/schemas/eventSchema'
+
+interface CommunityTabsProps {
+  navItems: string[]
+  description: string
+  events: EventCommunityType[]
+  members: UserType[]
+  organizers: OrganizationUserType[]
+}
+
+export default function CommunityTab({
+  navItems,
+  description,
+  events,
+  members,
+  organizers
+}: CommunityTabsProps) {
+  const [activeTab, setActiveTab] = useState<string>('About')
+  const [showAllContent, setShowAllContent] = useState<boolean>(false)
+
+  const lines = description.split('\n')
+  const visibleContent = showAllContent ? lines : lines.slice(0, 4)
+
+  const validEvents = events.filter(
```

Comment:
we should be filtering on frontend.

use the getPublic events function. If its not flexible enough you need to update that method.